host/src/lib/HelperFunctions.jsx

```diff
@@ -59,18 +78,8 @@ export const extractAnswers =  (teamsArray, currentState, currentQuestionId) =>
   teamsArray.forEach(team => {
     team.teamMembers && team.teamMembers.forEach(teamMember => {
       teamMember.answers && teamMember.answers.forEach(answer => {
-        if (answer.questionId === currentQuestionId) {
-          const isGameInPhaseOne = (
-            currentState === GameSessionState.CHOOSE_CORRECT_ANSWER ||
-            currentState === GameSessionState.PHASE_1_DISCUSS
-          ) && answer.isChosen;
-          const isGameInPhaseTwo = (
-            currentState === GameSessionState.PHASE_2_DISCUSS ||
-            currentState === GameSessionState.CHOOSE_TRICKIEST_ANSWER
-          ) && answer.isTrickAnswer;
-          if (isGameInPhaseOne || isGameInPhaseTwo) {
-            results.push({team, answer});
-          }
+        if (answer.questionId === currentQuestionId && isAnswerStateCurrent(answer.currentState, currentState)) {
```

Comment:
Checks currentState via `isAnswerStateCurrent` instead of using `isChosen` and `isTrickAnswer`